mkdocs/structure/pages.py

```diff
@@ -124,6 +124,7 @@ def active(self, value: bool):
 
     @property
     def is_index(self) -> bool:
+        """When `True`, indicates that this page is an index page, in the sense that it gets sorted to the beginning of the list of its siblings because its filename was `index.md` or something similar."""
```

Comment:
Feels a bit long for a summary.

```suggestion
        """When `True`, indicates that this page is an index page.
        
        Index pages get sorted to the beginning of the list of their siblings
        because their filename is `index.md` or `README.md`.
        """
```

Actually not entirely sure about `README.md`. I think @tomchristie would be able to confirm (and maybe rephrase to something better).

Reply:
Maybe just drop the second line in the body.